### DISCUSSION TITLE
Are the duplicated tvg-id's considered the same channel?

### DISCUSSION THREAD

**Discussion by tarrazuapps** [iptv-org/community#48] [Q&A] [answered]
There are a lot of duplicated tvg-ids in the US list. How should this be interpreted? Is that a mistake or should be considered multiple streams for the same channels?. I'm supposing is the later, but I do see different images for some of them which seem to imply that they are different channels.

**Comment by Dum4G** [comment, marked as answer]
Treat it as multiple streams for the same channels unless you see an obvious mistake. Various images is occurance of multiple people adding different sources at a different time. You can unify channel descriptions to avoid duplicates in generated playlists

**Reply by BellezaEmporium** [reply]
Wouldn't have said it better. Since there's lots of different streams for some channels (some even got over 6 different streams !) that could have been added over time, it's possible that some do not look at the other streams and simply add their own version.

**Comment by ilikenwf** [comment]
I really think we need a unique ID per channel beyond tvg-id that complies with the M3U spec, as managers can't differentiate between these channels based on URI, name, and tvg-id alone because of the duplication. Without some kind of ID, you end up with a duplicate entry every time a URL changes as a result.
